### DISCUSSION TITLE
Paths or routes with connections to other paths/routs

### DISCUSSION THREAD

**Discussion by rviktor85** [gdsfactory/gdsfactory#2105] [Q&A] [answered]
Hello, 

  Is there any elegant way in GDS Factory to connect a path or a route to a different path/route but not at the beginning or some specific location on the path/route itself? 
  
  For instance, it should look like this (but without a part that is rounded. 
  
<img width="351" alt="Screenshot 2023-09-20 at 11 28 48" src="https://github.com/gdsfactory/gdsfactory/assets/119857613/5d64dd5f-50c6-41a5-9950-fba17027b803">

**Comment by joamatab** [comment, marked as answer]
You can try define a T connector and connect to it
